frontend/src/scenes/constructor/components/ModalWindow/index.tsx

```diff
@@ -8,7 +8,11 @@ import {
 import { Button, Modal } from 'react-bootstrap';
 import { IAppState } from '../../../../common/models/store/IAppState';
 
-const ModalWindow:React.FC = () => {
+interface IProps {
+  id:string
+}
+
+const ModalWindow:FunctionComponent<IProps> = id => {
```

Comment:
Are you sure that this works? I think you forgot to destructure props 
```suggestion
const ModalWindow:FunctionComponent<IProps> = ({ id }) => {
```